contract/deploy.js

```diff
@@ -17,27 +17,31 @@ import bundleSource from '@endo/bundle-source';
 /**
  * @typedef {Object} DeployPowers The special powers that agoric deploy gives us
```

Comment:
I wish these were imported instead, but we have a bit of work on typing to do before we get there.